cmd/oteltestbedcol/go.mod

```
@@ -4,7 +4,7 @@ module github.com/open-telemetry/opentelemetry-collector-contrib/cmd/oteltestbed
 
 go 1.22.0
 
-toolchain go1.22.6
+toolchain go1.22.1
```

```suggestion
toolchain go1.22.6
```
We shouldn't be changing toolchain version here.